### ISSUE TITLE
Provide OSGI support

### ISSUE THREAD

**Issue by fabiolimace** [f4b6a3/ksuid-creator#8]

**Comment by fabiolimace** [comment]
Added OSGi entries to `MANIFEST.MF`:

```
Bundle-ManifestVersion: 2
Bundle-Name: ksuid-creator
Bundle-SymbolicName: com.github.f4b6a3.ksuid
Bundle-Version: 2.1.0
Export-Package: com.github.f4b6a3.ksuid
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

The OSGi symbolic name is the same as the root package name: `com.github.f4b6a3.ksuid`

**Comment by fabiolimace** [comment]
Released version 2.1.0.
